command/genesis/params.go

```diff
@@ -576,6 +583,27 @@ func (p *genesisParams) validateBurnContract() error {
 	return nil
 }
 
+func (p *genesisParams) validateGenesisBaseFeeConfig() error {
+	if p.baseFeeConfig == "" {
+		return errors.New("genesis base fee config must be set")
+	}
+
+	baseFeeInfo, err := parseBaseFeeConfig(p.baseFeeConfig)
+	if err != nil {
+		return err
+	}
+
+	if baseFeeInfo.baseFee <= 0 {
```

Comment:
Since it is uint64 it can't be less than 0. Same for `baseFeeInfo.baseFeeEM`. Plus `parseBaseFeeConfig()` would return an error if negative values are provided.